src/QMCHamiltonians/ECPotentialBuilder.cpp

```diff
@@ -63,15 +63,23 @@ bool ECPotentialBuilder::put(xmlNodePtr cur)
 #ifdef ENABLE_OFFLOAD
   NLPP_algo = "batched"; // set "batched" as the default
 #endif
+  std::string useDLA("no");
```

Comment:
for new code we should use the new naming conventions.  use_DLA

Reply:
+1

Reply:
Will fix it soon.